FirstProject/new-evaluation.html

```diff
@@ -0,0 +1,536 @@
+                            <option>0</option>
+                            <option>1</option>
+                            <option>2</option>
+                            <option>3</option>
+                        </select> <br>
+                            Issue Tracking <select>
+                            <option selected disabled>Evaluation</option>
+                            <option>0</option>
+                            <option>1</option>
+                            <option>2</option>
+                            <option>3</option>
+                        </select> <br>
+                        </li>
+                    </ul>
+            </fieldset>
+        </form>
+
+        <form class="language_form" style="text-align:center;">
+            <fieldset class="field_section" style="display:inline;">
+                <legend>HTTP</legend>
+                <ul style="display: inline-flex; text-align: right">
+                         <li>
+                            Protocol <select>
+                             <option selected disabled>Evaluation</option>
+                             <option>0</option>
+                             <option>1</option>
+                             <option>2</option>
+                             <option>3</option>
+                         </select> <br>
+                            Headers <select>
+                             <option selected disabled>Evaluation</option>
+                             <option>0</option>
+                             <option>1</option>
+                             <option>2</option>
+                             <option>3</option>
+                         </select> <br>
+                         </li>
+
+                        <li>
+                            Response codes <select>
+                            <option selected disabled>Evaluation</option>
+                            <option>0</option>
+                            <option>1</option>
+                            <option>2</option>
+                            <option>3</option>
+                        </select> <br>
+                            Request methods <select>
+                            <option selected disabled>Evaluation</option>
+                            <option>0</option>
+                            <option>1</option>
+                            <option>2</option>
+                            <option>3</option>
+                        </select> <br>
+                        </li>
+
+                        <li>
+                            REST <select>
+                            <option selected disabled>Evaluation</option>
+                            <option>0</option>
+                            <option>1</option>
+                            <option>2</option>
+                            <option>3</option>
+                        </select> <br>
+                            Sessions & Cookies <select>
```

Comment:
Hmm... yet another `&` on red background... What on Urf could be up with these?!?